tests/neg/implicitSearch.check

```diff
@@ -1,12 +1,7 @@
 -- [E172] Type Error: tests/neg/implicitSearch.scala:13:12 -------------------------------------------------------------
 13 |    sort(xs)  // error (with a partially constructed implicit argument shown)
    |            ^
-   |      No given instance of type Test.Ord[List[List[T]]] was found for parameter o of method sort in object Test.
-   |      I found:
-   |
-   |          Test.listOrd[List[T]](Test.listOrd[T](/* missing */summon[Test.Ord[T]]))
-   |
-   |      But no implicit values were found that match type Test.Ord[T].
+   |            No given instance of type Test.Ord[T] was found for parameter o of method listOrd in object Test
```

Comment:
The change in parameter name is good, but I think we should keep the "I found" part of the error message. Otherwise it's not clear why a listOrd is required in the first place. The "I found" part tells you that.